docs/configuration-reference/platforms/packet.md

```diff
@@ -221,7 +223,8 @@ node_type = var.custom_default_worker_type
 | `node_private_cidr`                   | Private IPv4 CIDR of the nodes used to allow inter-node traffic. Example "10.0.0.0/8"                                                                                                                                                                                             |        -        |    string    |   true   |
 | `enable_aggregation`                  | Enable the Kubernetes Aggregation Layer.                                                                                                                                                                                                                                          |      true       |     bool     |  false   |
 | `enable_tls_bootstrap`                | Enable TLS bootstraping for Kubelet.                                                                                                                                                                                                                                              |      true       |     bool     |  false   |
-| `network_mtu`                         | CNI interface MTU                                                                                                                                                                                                                                                                 |      1480       |    number    |  false   |
+| `encrypt_pod_traffic`                 | Enable in-cluster pod traffic encryption. If true `network_mtu` is reduced by 60 to make room for the encryption header.                                                                                                                                                          |      false      |     bool     |  false   |
```

Comment:
`network_mtu` stays the same right?  It's just that the `mtu` _inside_ the tunnel gets reduced by `60`

Reply:
Yes, value configured by the user stays the same. However, if you enable encryption, then it gets reduced by additional `40`, as IPIP overhead is `20` and wireguard encryption is `60`.

Reply:
got it. thanks